backend/eventd/silenced.go

```diff
@@ -118,7 +118,10 @@ func silencedBy(event *types.Event, silencedEntries []*types.Silenced) []string
 }
 
 func handleExpireOnResolveEntries(ctx context.Context, event *types.Event, store store.Store) error {
-	if !event.HasCheck() || !event.IsResolution() {
+	// Make sure we have a check that contains at least one silenced ID. Then,
+	// verify if the event is a resolution, which is a more expensive operation to
+	// perform
+	if !event.HasCheck() || len(event.Check.Silenced) == 0 || !event.IsResolution() {
```

Comment:
Ha, yes. I added this condition to prevent some tracing statements from appearing multiple times but completely forgot to remove that since it's definitely not an optimization.